dev/lint-python

```diff
@@ -99,6 +104,29 @@ else
     echo "flake8 checks passed."
 fi
 
+# Check python document style, skip check if pydocstyle is not installed.
```

Comment:
Nit: the indentation below looks different (4-space) from the rest of the script (2-space)

Reply:
Actually, seems like indentation is inconsistent in this file, so let's leave it for now